### PR TITLE
fix: Add support of ElementNotInteractable in assert_util

### PR DESCRIPTION
**Pull request by dkaras-splunk**

### CODE REVIEW

**Comment by dkaras-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA

File: tests/unit/test_base_test.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        (True, False, "is"),
+        (object, None, "is"),
+        (None, None, "is not"),
+        (1, 1, "is not"),
+        ("a", "a", "is not"),
+    ],
+)
+def test_operator_map_assert_error(ucc_mock, left, right, operator):
+    with pytest.raises(AssertionError):
+        assert ucc_mock.assert_util(left, right, operator) is None
```
Comment (uoboda-splunk) [inline tests/unit/test_base_test.py:154]: Maybe it is enoguh?: 
```suggestion
        ucc_mock.assert_util(left, right, operator)
```
`AssertionError` is raised inside function probably

File: tests/unit/test_base_test.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        (lambda x: x, lambda x: [], "in"),
+        (lambda x: x, lambda x: [x, 1, 2], "not in"),
+        (lambda x: x, lambda x: not x, "is"),
+        (lambda x: x, lambda x: x, "is not"),
+    ],
+)
+def test_operator_map_callable_obj_assert_error(ucc_mock, left, right, operator):
+    args = {"x": 1}
+    with pytest.raises(AssertionError):
+        assert ucc_mock.assert_util(left, right, operator, args, args) is None
```
Comment (uoboda-splunk) [inline tests/unit/test_base_test.py:66]: Maybe it is enoguh?:
```suggestion
        ucc_mock.assert_util(left, right, operator, args, args)
```
`AssertionError` is raised inside function probably

File: tests/unit/test_base_test.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
+from unittest.mock import MagicMock, patch
```
Comment (uoboda-splunk) [inline tests/unit/test_base_test.py:1]: Are these tests being run by CircleCi or GA?

**Comment by uoboda-splunk** [comment]
LGTM

**Review by uoboda-splunk — APPROVED** [review]

**Pull request merged**